resources/views/cart/overview.blade.php

```diff
@@ -33,7 +33,7 @@ class="mx-auto"
                     </div>
                 </div>
                 <div class="w-2/6 sm:w-1/6 lg:w-1/12 text-right pr-5">
-                    @{{ item.price | price }}
+                    @{{ $root.decideTax(item.price, item.price_excl_tax, 'cart_price') | price }}
```

Comment:
Maybe. It seemed extremely overkill to me to have the full price component for every little line just to get this exact same functionality. It'd be very verbose in the code and very inefficient, but it'd work. You'd still have to define the `cart_price` location code and essentially run the same function, but then within a component.

Reply:
How inefficient is the code? Considering on catalog & search pages it could be ran over 50 times

It would help readability and consistency to use or rename the component.
With it's name as a new user i would assume it would handle all price changes, and it would be confusing seeing it used in one place and replaced with raw js expressions somewhere else.
And something i can see coming in the future is the functionality of the price component and this function drifting apart as  the price component receives more features